docs/sphinx/sst/running_simeng_with_sst.rst

```diff
@@ -0,0 +1,361 @@
+through ``StandardInterface``, the cache will be connected to it. The default core to memory hierarchy component link provided by the installation are defined 
+below:
+
+.. code-block:: python
+    
+    cpu_to_cache_link = sst.Link("link1")
+    cpu_to_cache_link.connect((iface, "port", "100ps"), (l1cache, "high_network_0", "100ps"))
+    # or
+    cpu_to_cache_link = sst.Link("link1")
+    iface.addLink(cpu_to_cache_link, "port", "100ps")
+    l1cache.addLink(cpu_to_cache_link, "high_network_0", "100ps")
+
+Now that the SimEng core has been connected to the cache, the cache needs to connect to the lower levels of memory. For this configuration, it will be connected 
+to the memory controller otherwise it will be connected to lower levels of cache i.e. L2. As mentioned earlier the cache connects to lower levels through the 
+``low_network_0`` port.
+
+.. code-block:: python
+
+    l1_to_mem_link = sst.Link("link2")
+    l1_to_mem_link.connect((l1cache, "low_netowrk_0", "50ps"), (memctrl, "direct_link", "50ps"))
+    # or
+    l1_to_mem_link = sst.Link("link2")
+    l1cache.addLink(l1_to_mem_link, "low_network_0", "50ps")
+    memctrl.addLink(l1_to_mem_link, "direct_link", "50ps")
+
+.. warning::
+
+    The SimEng core YAML configuration file defines ``stack_size`` and ``heap_size`` parameters which are used to determine the size of the process image. 
+    In addition to these parameters, the process image size is also determined by the ELF binary header sections (which contain instructions and initialised data). To ensure 
+    the simulation is very fast, SimEng internally initialises the process image as a large char array that can index the highest virtual address encountered in 
+    the ELF binary header sections, eliminating the need for any address translation.
+
+    To make sure that the memory used by SST is consistent with the details mentioned above, the memory size of the backend i.e ``mem_size`` has been set to **2GiB** 
+    in the default SST ``config.py`` file. The ``SimengCoreWrapper`` checks if the SST memory backend has been configured with enough memory to store the process 
+    image. This check is done using the ``addr_range_end`` parameter of ``sstsimeng.simengcore``.
+
+    The user must also ensure that the maximum address accessible in the memory backend is consistent with ``addr_range_end`` parameter of the memory controller 
+    i.e ``memHierarchy.MemController`` and ``max_addr_range`` parameter of SimEng core i.e ``sstsimeng.simengcore``.
+
+.. note::
+   More examples of the SST ``config.py`` files can be found in the **SST-Elements** code base at `this <https://github.com/sstsimulator/sst-elements/tree/master/src/sst/elements/memHierarchy/tests>`_ 
+   path. Files starting with the prefix ``sdl`` contain different examples of memory hierarchy configurations which SST can simulate.
```

Comment:
The link should be presented as "... can be found in the SST-Elements code base, found `here`."